mars-sim-core/src/main/java/com/mars_sim/core/person/ai/task/meta/SleepMeta.java

```diff
@@ -53,17 +53,17 @@ public double getProbability(Person person) {
     	if (person.isOutside())
     		return 0;
     	
-    	if (person.isInSettlement()) {
-	        Building b = person.getBuildingLocation();
-	        if ((b != null) && (b.getCategory() == BuildingCategory.EVA_AIRLOCK)) {
-		        boolean inZone0 = b.getEVA().getAirlock().isInZone(person, AirlockZone.ZONE_0);
-		        if (!inZone0)
-		        	// if a person is in zone 0 at the inner door trying to ingress,
-		        	// and at the same time very fatigue, 
-		        	// then it's okay to allow this person to fall asleep.
-		        	return 0;
-	        }
-    	}
+//    	if (person.isInSettlement()) {

```

Comment:
Basically we always calculate a sleep probability, fine.
Can you delete the commented out code. It is always possible to find by using git history.